docker/grouper/Dockerfile

```diff
@@ -76,6 +77,14 @@ RUN apk add --update curl \
     && mv "apache-tomcat-$TOMCAT_VERSION" "$CATALINA_HOME" \
     && cd "$CATALINA_HOME" \
     && rm bin/*.bat \
+    && mkdir -p "$TOMCAT_CONF" \
+    && mv "$CATALINA_HOME/conf/server.xml" "$TOMCAT_CONF/server.xml" \
+    && ln -s "$TOMCAT_CONF/server.xml" "$CATALINA_HOME/conf/server.xml" \
+    && mv "$CATALINA_HOME/conf/tomcat-users.xml" "$TOMCAT_CONF/tomcat-users.xml" \
+    && ln -s "$TOMCAT_CONF/tomcat-users.xml" "$CATALINA_HOME/conf/tomcat-users.xml" \
+    && rm -f "$CATALINA_HOME/bin/setenv.sh" \
+    && touch "$TOMCAT_CONF/setenv.sh" \
+    && ln -s "$TOMCAT_CONF/setenv.sh" "$CATALINA_HOME/bin/setenv.sh" \
```

Comment:
Correct. I guess I could have included a default Grouper user in `tomcat-users.xml`, but I figured that these files will be overridden most of the time.